README.md

```diff
@@ -3,9 +3,38 @@
 [![Build status](https://badge.buildkite.com/c9edf020a4aec976f9835e54751cc5409d843adbb66d043bd3.svg?branch=main)](https://buildkite.com/ankitects/anki-ci)
 
 This repo contains the source code for the computer version of
-[Anki](https://apps.ankiweb.net).
+[Anki](https://apps.ankiweb.net) for people who would like to help improve the code.
+
+# About
+
+[Anki](https://apps.ankiweb.net) helps you learn faster with user created flashcards that makes learning easier and more efficient compared to traditional study methods.
+
+Card content is endless as they support:
+
+- images :framed_picture:
+- audio :loud_sound:
+- videos :film_strip:
+- scientific markup (Via LaTeX) :dna:
+
```

Comment:
Can we cut out the extra text here and just say something like "Anki is a spaced repetition program. Please see [the website](...) to learn more."? The emoji is a little more playful than the style we typically use, and "they support" sounds bad.

Reply:
Extra text removed and added recommended statement  describing Anki.